ecr/ref.go

```diff
@@ -70,6 +74,60 @@ func (spec ECRSpec) Registry() string {
 	return spec.arn.AccountID
 }
 
+// ParseEcrImageNameToRef takes an ECR image name and returns its canonical `ref` format.
+// i.e. ecr.aws/arn:<partition>:ecr:<region>:<account>:repository/<name>:<tag>
+func ParseEcrImageNameToRef(input string) (string, error) {
```

Comment:
nit: In Go, [initialisms should be capitalized](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).  Can you capitalize "ECR"?